hadongun/큐/card.cpp

```diff
@@ -0,0 +1,48 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+
+int implement(int card[], int N);
+
+int main(void)
+{
+    int N = 0;
+    int card[500001] = { 0 };
+    scanf("%d", &N);
+    for (int i = 0; i < N; i++)
+    {
+        card[i] = i + 1;
+    }
+
+    printf("%d", implement(card, N));
+
+}
+
+int implement(int car[], int k)
+{
+    int first_card;
+    int last_card;
+    int last_index;
+    int moving_box;
+    int saving_box;
+    int temp_1 = 0;
+    int temp_2 = 0;
```

Comment:
```suggestion
    int saving_box;
```
사용하지 않는 변수가 너무 많은 것 같아요! 불필요한 코드를 지우는 것 만으로도 코드의 가독성을 높일 수 있으니 정리하면서 코드를 짜는 걸 추천 드립니다!!